Makefile

```diff
@@ -12,14 +12,27 @@ IMAGE_TAG ?= latest
 OPERATOR_IMAGE ?= cluster-network-addons-operator
 REGISTRY_IMAGE ?= cluster-network-addons-registry
 
+fmt: goimports whitespace
+
+goimports:
+	goimports -w ./pkg ./cmd
+
+whitespace:
+	./hack/whitespace.sh --fix
+
+check: whitespace-check vet goimports-check generate-api-check test
+
+whitespace-check:
+	./hack/whitespace.sh
+
 vet:
 	go vet ./pkg/... ./cmd/...
 
-fmt:
-	go fmt ./pkg/... ./cmd/...
+goimports-check:
+	goimports -d ./pkg ./cmd
 
 test:
-	go test -v -race ./pkg/... ./cmd/... -coverprofile cover.out 
+	go test -v -race ./pkg/... ./cmd/... -coverprofile cover.out
 
 docker-build: docker-build-operator docker-build-registry
```

Comment:
now here we can add all the checks and api generation and it's not going to take much time since we are tracking source files.

Reply:
Done